blog/2024-09-19-update-version-compare.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Please update the filename to today's date.

Reply:
Done